gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/service/ConfidentialityServiceTest.java

```diff
@@ -13,8 +13,10 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
+import static uk.nhs.adaptors.pss.translator.util.MetaFactoryUtil.assertMetaSecurityIsNotPresent;
 
 class ConfidentialityServiceTest {
+
     private static final String DUMMY_PROFILE = "Test-MyProfile-1";
     private static final String DUMMY_PROFILE_URI = "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-%s".formatted(DUMMY_PROFILE);
```

Comment:
I understand this is out-of-scope of this PR, though just wanted to raise! Technically this is a `URL` and not a `URI` 😄 

```suggestion
    private static final String DUMMY_PROFILE = "Test-MyProfile-1";
    private static final String DUMMY_PROFILE_URL = "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-%s".formatted(DUMMY_PROFILE);
```